src/parser/jobs/drg/modules/Buffs.js

```diff
@@ -190,19 +189,36 @@ export default class Buffs extends Module {
 			</Trans>,
 		}))
 
-		const badlyTimedLcs = this._buffWindows[STATUSES.LANCE_CHARGE.id].history.filter(window => window.casts.length > 0 && !LC_FIRST_ACTIONS.includes(window.firstGcd)).length
+		const badLanceCharges = this._buffWindows[STATUSES.LANCE_CHARGE.id].history.filter(window => window.casts.length > 0 && window.isBad).length
 		this.suggestions.add(new TieredSuggestion({
 			icon: ACTIONS.LANCE_CHARGE.icon,
 			content: <Trans id="drg.buffs.suggestions.bad-lcs.content">
-				Avoid using <ActionLink {...ACTIONS.LANCE_CHARGE}/> immediately before any GCD other than <ActionLink {...ACTIONS.DISEMBOWEL}/>, <ActionLink {...ACTIONS.CHAOS_THRUST}/>, and <ActionLink {...ACTIONS.FULL_THRUST}/> in order to get the most possible damage out of each window.
+				Avoid using <ActionLink {...ACTIONS.LANCE_CHARGE}/> immediately after <ActionLink {...ACTIONS.CHAOS_THRUST}/> or <ActionLink {...ACTIONS.FULL_THRUST}/> in order to get the most possible damage out of each window.
 			</Trans>,
 			tiers: {
 				1: SEVERITY.MINOR,
 				3: SEVERITY.MEDIUM,
 			},
-			value: badlyTimedLcs,
+			value: badLanceCharges,
 			why: <Trans id="drg.buffs.suggestions.bad-lcs.why">
-				{badlyTimedLcs} of your Lance Charge windows started on a non-optimal GCD.
+				{badLanceCharges} of your Lance Charge windows started right after a standard combo finisher.
+			</Trans>,
+		}))
+
+		// I'm not going to say how close I came to naming this variable badDragons
```

Comment:
I'm almost but not quite entirely disappointed in you. And I'm not going to say in which direction.